src/models/db.js

```diff
@@ -231,6 +231,34 @@ export default config => {
     });
   };
 
+  const getFilter = parsedFilter => {
```

Comment:
Could we move this to the query parser so req.odata.$filter has the form that we want at this point?